pkg/virt-controller/services/multus_annotations.go

```diff
@@ -69,12 +71,40 @@ func GenerateMultusCNIAnnotationFromNameScheme(namespace string, interfaces []v1
 		}
 	}
 
+	if config.NetworkBindingPlugingsEnabled() {
+		for _, iface := range interfaces {
+			if iface.Binding != nil {
+				podInterfaceName := networkNameScheme[iface.Name]
+				bindingPluginAnnotationData, err := newBindingPluginMultusAnnotationData(
+					config.GetConfig(), iface.Binding.Name, namespace, podInterfaceName)
+				if err != nil {
+					return "", err
+				}
+				multusNetworkAnnotationPool.add(*bindingPluginAnnotationData)
```

Comment:
> I dont think there is difference as long as the "base" CNI invoked before the plugin CNI, right after the "base" or after other interfaces CNIs

There is a difference: Resources will not be allocated for the next interface if the current injected CNI fails to configure the pod. It will also allow potential collisions to be detected in the correct order (in case the binding CNI/s touch some shared thing).